emmo/.htaccess

```diff
@@ -0,0 +1,99 @@
+Options +FollowSymLinks
+
+# Rewrite engine setup
+RewriteEngine On
+
+
+# Rule 1: https://w3id.org/emmo
+#
+# If the request appears coming from a browser, redirect to the html documentation
+RewriteCond %{HTTP_USER_AGENT} ^Mozilla/
+RewriteRule ^$ https://emmo-repo.github.io/emmo.html [R=303,NE,L]
+#
+# ...otherwise redirect to inferred/squashed ontology
+RewriteRule ^emmo-inferred$ https://emmo-repo.github.io/emmo-inferred.ttl [R=303,NE,L]
+RewriteRule ^$ https://emmo-repo.github.io/emmo.ttl [R=303,NE,L]
+
+
+# Rule 2: https://w3id.org/emmo/
+#
+# emmo.ttl file in the root of the master branch
+RewriteRule ^/$ https://raw.githubusercontent.com/emmo-repo/EMMO/master/emmo.ttl [R=303,NE,L]
+
+
+# Rule 3: https://w3id.org/emmo/{VERSION}
+#
+# If the request appears coming from a browser, redirect to the html documentation of given version
+RewriteCond %{HTTP_USER_AGENT} ^Mozilla/
+RewriteRule ^([0-9][^/]+)$ https://emmo-repo.github.io/versions/$1/emmo.html [R=303,NE,L]
+#
+# ...otherwise redirect to inferred/squashed ontology
+RewriteRule ^([0-9][^/]+)-inferred$ https://emmo-repo.github.io/versions/$1/emmo-inferred.ttl [R=303,NE,L]
+RewriteRule ^([0-9][^/]+)$ https://emmo-repo.github.io/versions/$1/emmo.ttl [R=303,NE,L]
+
+
+
+# Rule 4: https://w3id.org/emmo/{VERSION}/
+#
+# emmo.ttl file in the root of branch for the given version
+RewriteRule ^([0-9][^/]+)/$ https://raw.githubusercontent.com/emmo-repo/EMMO/$1/emmo.ttl [R=303,NE,L]
+
+
+# Rule 5: https://w3id.org/emmo/{PATH}/{MODULE}
+#
+# Turtle file for given module of EMMO
+RewriteRule ^([^0-9].+)/?$ https://raw.githubusercontent.com/emmo-repo/EMMO/$1.ttl [R=303,NE,L]
```

Comment:
please note that this condition will also evaluate all `/domain-.../` paths as fulfilled => needs to be placed after all domain rules

Reply:
Good point. I swapped the domain ontology section with EMMO-specific redirections.